inotify.lisp

```diff
@@ -1,5 +1,12 @@
 (in-package #:org.shirakumo.file-notify)
 
+;; FreeBSD has Linux compatibility layer in libinotify.so available via ports.
+#+freebsd
+(progn
+  (cffi:define-foreign-library libinotify
+    (:freebsd (:or "libinotify.so")))
+  (cffi:use-foreign-library libinotify))
```

Comment:
Please only use foreign libraries in `init`. Loading them early complicates deployment.